README.md

```diff
@@ -78,3 +78,32 @@ start the Bee cluster
 ```sh
 $ ./scripts/environment.sh start
 ```
+
+## Utilities
+
+It is possible to generate random traffic in your cluster:
+
+```sh
+$ npm run gen:traffic 
+```
+
+The script is in an infinite loop, so if you want to stop the generation you have to terminate it manually in your terminal by pressing `Ctrl^C`.
+
+If you don't specify any parameters it will produce 400 chunks/0.5 sec that the script tries to upload on the `http://localhost:1633` - that is the binded port of the queen node if you orchestrated the environment with the `envrionment.sh`.
+
+The following way you can pass parameters
+
+1. HOST - Host that has reachable port to the Bee API. [string,Default:'localhost']
+2. SECURE_CONNECTION - whether the connection is HTTPS. [boolean,Default:'false']
```

Comment:
I find this very strange. Why do we need to know this information explicitly?

Reply:
well, this is how I construct the url, it defines the http or https protocol that's going to be used

Reply:
Ok I understand that. But explicitly there is no difference between interacting with Bee node that is behind ssl and one that is not, right? I'd advocate to remove these and let users to just pass in the bee URLs as array. 

My reasoning:
1. It gives more flexibility (e.g. running bee nodes on different hosts or under path)
2. It makes the usage much simpler, you just pass in the URLs and don't need to think about it
3. It makes the code much simpler. You just iterate over the URLs and don't need to construct them

Reply:
See my comment below